### ISSUE TITLE
Additional Features

### ISSUE THREAD

**Issue by JWynegar** [konnected-io/noonlight-hass#16]
Looking at the Hubitat version of the integration it looks like there is a way to send event data to dispatchers using the v1 API. Can we add this to the Home Assistant version?

**Comment by KruseLuds** [comment]
I agree 1,000 percent! In my build out of an alarm system, i have created a set of response messages I would LOOOVE to be passed on with an alarm being triggered that is very specifically complaining what was triggered and where...
